### ISSUE TITLE
IPS alerting disabled rules

### ISSUE THREAD

**Issue by mnaiman** [opnsense/core#4725]
OPNsense 21.1.1 (amd64, OpenSSL).

I have several policies which disable several rules.
After fixing https://github.com/opnsense/core/issues/4695#issuecomment-777745841 they are properly disabled.

Wierd is that SOME of disabled rules (not all) ale still reported on alerts page, why?

Picture is self explanatory.
Both selected rules are NOT enabled, still there are reported.
Lot of other disabled rules are triggered and not alerted properly.

![image](https://user-images.githubusercontent.com/2264442/108591884-3bfc0880-736b-11eb-8ccb-b94d54d33afb.png)

Thanks

**Comment by AdSchellevis** [comment]
accidentally patched the already working version? screenshot doesn't look like it belongs to 21.1.1

**Comment by mnaiman** [comment]
![image](https://user-images.githubusercontent.com/2264442/108593607-ecbad580-7374-11eb-9c5f-3942b19c1bfd.png)

**Comment by mnaiman** [comment]
Followed only official upgrade path 20.1-20.7-21.1 etc + applied patch from your post https://github.com/opnsense/core/issues/4695#issuecomment-777345538

**Comment by AdSchellevis** [comment]
patch is already included in 21.1.1, adding it again will revert...

**Comment by mnaiman** [comment]
no its not :/
Right now I have reinstalled opnsense package and bug in https://github.com/opnsense/core/issues/4695#issuecomment-777345538 re-appeared.

**Comment by AdSchellevis** [comment]
checked again, patch is included in 21.1.1, which doesn't mean you don't have another issue, but applying the patch again certainly won't help.

**Comment by mnaiman** [comment]
I don't get it.
Im on 21.1.1. 
I did audit health check, all files are ok.

Matching by classtype is not working as described in https://github.com/opnsense/core/issues/4695

So I ran opnsense-patch b465a41

```
| src/opnsense/scripts/suricata/lib/rulecache.py | 9 ++++++++-
| 1 file changed, 8 insertions(+), 1 deletion(-)
|
|diff --git a/src/opnsense/scripts/suricata/lib/rulecache.py b/src/opnsense/scripts/suricata/lib/rulecache.py
|index 85e02739e7..7bd3754aca 100755
|--- a/src/opnsense/scripts/suricata/lib/rulecache.py
|+++ b/src/opnsense/scripts/suricata/lib/rulecache.py
--------------------------
Patching file opnsense/scripts/suricata/lib/rulecache.py using Plan A...
Hunk #1 succeeded at 82.
```

Reappling policies. All are matching ok, even classtypes.

Audit healthcheck
```
Checking all packages: ....
opnsense-21.1.1: checksum mismatch for /usr/local/opnsense/scripts/suricata/lib/rulecache.py
```

**Comment by mnaiman** [comment]
I have downloaded these source codes and rulecache.py is without patch https://github.com/opnsense/core/releases/tag/21.1.1

**Comment by AdSchellevis** [comment]
@mnaiman ok, now I'm mixing two different patches, https://github.com/opnsense/core/commit/b465a418a0f9e17b7f73cc9dafb1cb4571cd6fb7 is indeed not included (should be in 21.1.2) the other patches are in. with the details provided I don't think we can track this further, sid 29456 seems to be some rule from snort, details about the environment might matter.... we offer issue templates to streamline the proces, it's probably better to close this issue and add a new one using our templates and proving more details about how one could reproduce your issue.
